x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/AutoFollowCoordinator.java

```diff
@@ -304,10 +304,11 @@ private void followLeaderIndex(String clusterAlias,
             final String leaderIndexName = indexToFollow.getName();
             final String followIndexName = getFollowerIndexName(pattern, leaderIndexName);
 
-            String leaderIndexNameWithClusterAliasPrefix = clusterAlias.equals("_local_") ? leaderIndexName :
-                clusterAlias + ":" + leaderIndexName;
             ResumeFollowAction.Request request = new ResumeFollowAction.Request();
-            request.setLeaderIndex(leaderIndexNameWithClusterAliasPrefix);
+            if ("_local_".equals(clusterAlias) == false) {
+                request.setLeaderClusterAlias(clusterAlias);
```

Comment:
This is an example where I think this should match the body field `leader_cluster`.